.github/workflows/rust.yml

```diff
@@ -95,7 +90,7 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: "lts/*"
```

Comment:
for wasm builds, latest LTS should be fine and this removes a version string that needs to be kept up to date